modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonAbstractConnexionServerCodegen.java

```diff
@@ -165,6 +167,8 @@ public PythonAbstractConnexionServerCodegen(String templateDirectory, boolean fi
                 defaultValue("8080"));
         cliOptions.add(CliOption.newBoolean(USE_NOSE, "use the nose test framework").
                 defaultValue(Boolean.FALSE.toString()));
+        cliOptions.add(CliOption.newBoolean(SRC_LAYOUT, "put python module under src/").
```

Comment:
@cognifloyd would there ever be a use case where we would want to name the folder something other than src?
If so why don't we make this parameter more general like sourceFolderName type string.
What do you think?

Reply:
`src/` is pretty standard, but I suppose I have seen projects that do `lib/` as well.
So, an empty string (or maybe a `.`) means no subdir and a string == a subdir? hmm.

Reply:
hmm. Here's [someone](https://stackoverflow.com/questions/30461071/configuring-the-directories-that-nosetests-looks-into) that puts his source in a `python` dir. So, there may be use cases where that would be helpful. Like in multi-language projects or where you include client and server libs in the same repo.

So, the name `srcLayout` only makes sense as a boolean. What would an option be for selecting an alternate directory for the python source files? `pythonSrcRoot`? `srcLayoutDir`? or would you do two vars, one for the dir to use and a boolean to enable it?

Reply:
How about the string pythonSrcRoot? And make it an empty value by default.

Reply:
done